apps/elasticsearch.go

```diff
@@ -105,7 +105,7 @@ func (p LoggingProcessorElasticsearchJson) Components(tag, uid string) []fluentb
 	c := []fluentbit.Component{}
 
 	// sample log line:
-	// {"type": "server", "timestamp": "2022-01-17T18:31:47,365Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "elasticsearch", "node.name": "ubuntu-impish", "message": "initialized" }
+	// {"type": "server", "timestamp": "2022-01-17T18:31:47,365Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "elasticsearch", "node.name": "ubuntu-jammy", "message": "initialized" }
```

Comment:
I forgot I did this, it was kind of a joke to myself but I guess I'll just leave it :laughing: